PetAdoption-iOS/PetAdoption-iOS/Controllers/filterViewControlelr.swift

```diff
@@ -0,0 +1,135 @@
+//
+//  filterViewControlelr.swift
```

Comment:
This is probably a little nitpicky, but the file name is, for one, misspelled (`Controlelr` instead of `Controller`), and two, convention is to capitalize the names of the files.

Reply:
Good eyes :)
fixed with 31da2d4